src/main/java/plugily/projects/villagedefense/arena/Arena.java

```diff
@@ -395,11 +394,11 @@ public ScoreboardManager getScoreboardManager() {
    * @return list of spawned zombies in arena
    */
   @NotNull
-  public List<Zombie> getZombies() {
+  public List<Creature> getZombies() {
```

Comment:
Please deprecate this method and create a new getter/setter method with an appropriate name.